### ISSUE TITLE
RStudio hangs on opening certain project

### ISSUE THREAD

**Issue by woodwards** [rstudio/rstudio#2432]
### System details

    RStudio Edition : Desktop
    RStudio Version : Version 1.1.442 
    OS Version      : Windows 10 Enterprise 1607 14393.2068
    R Version       :  R version 3.4.4 (2018-03-15) -- "Someone to Lean On"

### Steps to reproduce the problem
Open bopsoil.Rproj file either by double clicking in Windows Explorer or from File menu in RStudio.

### Describe the problem in detail
RStudio opens Console and R start message appears, Environment and File windows remain greyed out, Console accepts input but does not return answer to > 2+2

### Describe the behavior you expected
Project should open. Other projects open normally.

I tried 
C:\Users\WoodwardS\Documents\Projects\Pasture_Potential\bopsoil>mv .Rproj.user .Rproj.user.backup
C:\Users\WoodwardS\AppData\Local>mv rstudio-desktop rstudio-desktop-backup

but this did not resolve the issue. The project in question is a Shiny App, which is deployed at
https://woodward-dairynz.shinyapps.io/bysoil4/

**Comment by kevinushey** [comment]
Does your project have a `.git` directory? Is that `.git` directory a hidden folder?

**Comment by woodwards** [comment]
Yes it does. Although I have unhidden the hidden folders.

**Comment by kevinushey** [comment]
Does moving the `.git` directory away temporarily help? Or, does forcing it to be a hidden directory again resolve the issue?

**Comment by woodwards** [comment]
Yes! Hiding the `.git` directory seems to have resolved the issue. Thanks!

**Comment by woodwards** [comment]
Hmm, git won't let me make a commit.

```
>>> git commit -F C:/Users/WOODWA~1/AppData/Local/Temp/RtmpuUTryR/git-commit-message-2584629112fc.txt
fatal: could not open '.git/COMMIT_EDITMSG': Permission denied
```

**Comment by woodwards** [comment]
Unhiding the git dir allowed me to commit.

**Comment by woodwards** [comment]
Solution: I had to delete `.git\COMMIT_EDITMSG`

**Comment by jmcphers** [comment]
This issue is tracked in https://github.com/rstudio/rstudio/issues/2141, and has now been fixed in RStudio 1.1.444.

https://www.rstudio.com/products/rstudio/download/preview/

**Comment by woodwards** [comment]
Yep seems to be resolved.
